pkg/builtin/manifest.yml

```diff
@@ -1918,3 +1918,15 @@ extensions:
               title: Layer
               type: ref
               ui: layer
+  - id: timeline
+    type: widget
+    name: Timeline
+    description: Operate timeline for earth.
```

Comment:
"A timeline widget that allows for viewing layers and data at specific points in time"

Reply:
This was fixed by [fix: improve description](https://github.com/reearth/reearth-backend/pull/161/commits/43413d48f2dbea9f6a2dd637770ace5a8fef3542).
@rot1024 Could you review Japanese translation?